scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -5,9 +5,8 @@ package parsers
 import scala.language.implicitConversions
 import scala.compat.Platform.EOL
 import scala.reflect.{ClassTag, classTag}
-import scala.runtime.ScalaRunTime
-import scala.collection.{ mutable, immutable }
-import mutable.{ ListBuffer, StringBuilder }
+import scala.collection. mutable 
```

Comment:
This is a known issue https://github.com/scalacenter/scalafix/issues/216 due to be fixed in the next release.